layers/+emacs/org/packages.el

```diff
@@ -951,7 +951,10 @@ Headline^^            Visit entry^^               Filter^^                    Da
         "ra" 'org-roam-alias-add))
     :config
     (progn
-      (spacemacs|hide-lighter org-roam-mode))))
+      (spacemacs|hide-lighter org-roam-mode)
+      (if org-enable-roam-protocol
+          (add-hook 'org-roam-mode-hook (lambda ()
+                                          (use-package org-roam-protocol)))))))
```

Comment:
Why not `(require 'org-roam-protocol)`?

Reply:
Yeah, I guess `use-package` doesn't provide any advantage over a plain require in this situation. I feel like I read somewhere in the Spacemacs docs that `use-package` should be prefered, but that was probably meant in some other context.

Reply:
Yes. It did say in general you should not use `require` because spacemacs deliberately want to avoid the dependencies between packages.

But `require` is much simpler.